code/modules/atmospherics/machinery/components/unary_devices/cryo.dm

```diff
@@ -383,6 +402,9 @@
 /obj/machinery/atmospherics/components/unary/cryo_cell/close_machine(mob/living/carbon/user, density_to_set = TRUE)
 	treating_wounds = FALSE
 	if((isnull(user) || istype(user)) && state_open && !panel_open)
+		if(loc == usr.loc)
+			to_chat(usr, span_warning("You can't close [src] on yourself!"))
```

Comment:
```suggestion
		if(loc == user.loc)
			to_chat(user, span_warning("You can't close [src] on yourself!"))
```

Reply:
I had it as this initially, but was getting an error where `user` was always null. I'll try it again but if it still doesn't work, thoughts on how to fix that?